solr/core/src/java/org/apache/solr/response/SolrQueryResponse.java

```diff
@@ -61,7 +64,7 @@
  */
 public class SolrQueryResponse {
   public static final String NAME = "response";
-  public static final String RESPONSE_HEADER_PARTIAL_RESULTS_KEY = CommonParams.PARTIAL_RESULTS;
```

Comment:
Originally I did this on purpose to avoid getting these two strings out of sync.

Reply:
Right but they aren't actually the same string. One is a request parameter and the other is a response header key. The fact they are textually similar is a coincidence (and packages now make this difficult)